packages/webvh/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@credo-ts/webvh",
+  "version": "0.4.0",
```

Should be synced with the other packages

```suggestion
  "version": "0.5.13",
```